go.mod

```diff
@@ -165,20 +164,20 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/apiextensions-apiserver v0.0.0 // indirect
-	k8s.io/apiserver v0.32.0-alpha.0 // indirect
-	k8s.io/component-helpers v0.32.0-alpha.0 // indirect
-	k8s.io/controller-manager v0.32.0-alpha.0 // indirect
-	k8s.io/kms v0.32.0-alpha.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20240730131305-7a9a4e85957e // indirect
+	k8s.io/apiserver v0.32.0 // indirect
+	k8s.io/component-helpers v0.32.0 // indirect
+	k8s.io/controller-manager v0.32.0 // indirect
+	k8s.io/kms v0.32.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20241105132330-32ad38e42d3f // indirect
 	k8s.io/kubectl v0.0.0 // indirect
-	k8s.io/kubelet v0.32.0-alpha.0 // indirect
-	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.30.3 // indirect
-	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
-	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
+	k8s.io/kubelet v0.32.0 // indirect
+	sigs.k8s.io/apiserver-network-proxy/konnectivity-client v0.31.0 // indirect
+	sigs.k8s.io/cloud-provider-azure/pkg/azclient/cache v0.2.0 // indirect
+	sigs.k8s.io/json v0.0.0-20241010143419-9aa6b5e7a4b3 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.4.2 // indirect
 )
 
 replace (
-	github.com/google/cel-go => github.com/google/cel-go v0.17.8
```

Comment:
you also need to change the k8s version(replacing v0.30.3 with v0.32.0) below

Reply:
bumped k8s.io to 0.32